src/App.js

```diff
@@ -1,25 +1,26 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Route, Routes } from "react-router-dom";
+import Header from "./components/Header";
+import Homepage from "./components/Homepage";
+import Login from "./components/Login";
+import Reviews from "./components/Reviews";
+import SingleReview from "./components/SingleReview";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <div className="App">
+            <Header />
+            <Routes>
+                <Route path="/" element={<Homepage />}></Route>
```

Comment:
YOur home page isn't rendering anything at the moment and there's no way to navagate to the reviews. The route which renders the reviews looks good. But have a think about what the landing page is going to look like.